cargo-powdr/src/main.rs

```diff
@@ -0,0 +1,137 @@
+//! The cargo-powdr project creator.
+
+use clap::{CommandFactory, Parser, Subcommand};
+
+use std::{
+    fs,
+    io::{self, Write},
+    path::Path,
+};
+
+#[derive(Parser)]
+#[command(name = "cargo-powdr", author, version, about, long_about = None)]
+struct Cli {
+    #[command(subcommand)]
+    command: Option<Commands>,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    /// Create a new powdr project.
+    New {
+        /// The name of the project
+        name: Option<String>,
+
+        /// The name of the guest
+        #[arg(short, long)]
+        #[arg(default_value_t = String::from("my-powdr-guest"))]
```

Comment:
maybe just `powdr-guest` as a default? most likely this name wont be changed unless people are doing something more complicated like having more than one guest in the same project?